package.json

```diff
@@ -45,7 +45,7 @@
     "ammo-debug-drawer": "^0.1.0",
     "cannon-es": "^0.9.1",
     "patch-package": "^6.5.0",
-    "three-to-ammo": "^1.0.1",
+    "@c-frame/three-to-ammo": "^1.0.3",
```

Comment:
I just published 1.0.2.  Not sure what 1.0.3 is?

Reply:
OK - just seen https://github.com/c-frame/three-to-ammo/pull/5 & now understand.  I will merge & pubish that and then come back to this.